packages/create-react-app/index.js

```diff
@@ -152,10 +152,10 @@ function getInstallPackage(version) {
 
 // Extract package name from tarball url or path.
 function getPackageName(installPackage) {
-  if (~installPackage.indexOf('.tgz')) {
+  if (installPackage.indexOf('.tgz') > 0) {
     return installPackage.match(/^.+\/(.+)-.+\.tgz$/)[1];
-  } else if (~installPackage.indexOf('@')) {
-    return installPackage.split('@')[0];
+  } else if (installPackage.indexOf('@') > 0) {
```

Comment:
This needs a comment because somebody else might change it to `-1` later without understanding the reason.